From f35ecb4b8622a50f08795f4791019e9dec66df97 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Sat, 16 Mar 2019 08:36:28 -0700
Subject: [PATCH] Fix build in Clear

get_config_var("libdir") returns the path where the Python modules are
located, not where the library itself is. (/usr/lib/python3.7 vs
/usr/lib64/libpython3.7m.so.1.0)
---
 build_scripts/main.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build_scripts/main.py b/build_scripts/main.py
index 05e9b0f7..f682acef 100644
--- a/build_scripts/main.py
+++ b/build_scripts/main.py
@@ -515,7 +515,7 @@ class PysideBuild(_build):
         py_executable = sys.executable
         py_version = "{}.{}".format(sys.version_info[0], sys.version_info[1])
         py_include_dir = get_config_var("INCLUDEPY")
-        py_libdir = get_config_var("LIBDIR")
+        py_libdir = "/usr/lib64"
         py_prefix = get_config_var("prefix")
         if not py_prefix or not os.path.exists(py_prefix):
             py_prefix = sys.prefix
-- 
2.23.0

